operations/rates.md

```diff
@@ -15,10 +15,18 @@ Returns all rates \(pricing setups\) and rate groups \(condition settings\) of t
     "Client": "Sample Client 1.0.0",
     "ServiceIds": [
         "bd26d8db-86da-4f96-9efc-e5a4654a4a94"
+    ], 
+    "RateIds": [
+        "ed4b660b-19d0-434b-9360-a4de2ea42eda"
     ],
+    "UpdatedUtc": {
+        "StartUtc": "2022-10-15T00:00:00Z",
+        "EndUtc": "2022-10-20T00:00:00Z"
+    },
     "Extent": {
         "Rates": true,
-        "RateGroups": true
+        "RateGroups": true,
+        "RateRestrictions": false
```

Comment:
RateRestrictions are deprecated, so maybe it's better not to include them in the example.

Reply:
Somehow I managed to missed that it's deprecated... that is why I'm more for not removing but marking as deprecated.